README.md

```diff
@@ -10,37 +10,159 @@ This is a port of [git-machete](https://github.com/VirtusLab/git-machete#git-mac
 
 This plugin is available on [JetBrains Marketplace](https://plugins.jetbrains.com/plugin/14221-git-machete).
 
+**It is compatible with all JetBrains products except Android Studio.**
+
 ![](docs/demo-workflow.gif)
 
 
-## Logging
+## How it works
 
-SLF4J logging in this plugin has the following categories:
+Git Machete IntelliJ Plugin is a port of a great console tool - [git-machete](https://github.com/VirtusLab/git-machete#git-machete) - into an IntelliJ plugin.
+Git Machete can help you manage your repository branches and keep it in sync with each other and its counterparts on remote repository.
+Let's see how this plugin can help you:
 
-```
-binding
-branchlayout
-gitcore
-gitmachete.backend
-gitmachete.frontend.actions
-gitmachete.frontend.externalsystem
-gitmachete.frontend.graph
-gitmachete.frontend.ui
-```
+### Where to find this plugin window
 
-The standard practice in Java logging is to use fully-qualified name of each class as category;
-in our case, however, we're only ever using the categories provided above.
-FQCN (and method name), however, is always a part of the log message itself.
+Git Machete IntelliJ Plugin is available under `Git` section (in the bottom of the IDE) in the `Git Machete` tab.
+You can also use `Ctrl` + `Alt` + `Shift` + `M` shortcut to open this plugin.
 
-By default, IntelliJ logs everything with level `INFO` and above into `idea.log` file. <br/>
-The exact location depends on a specific IntelliJ installation; check `Help` -> `Show Log in Files` to find out. <br/>
-Tip: use `tail -f` to watch the log file as it grows.
+![](docs/open_git_machete.gif)
 
-To enable logging in `DEBUG` level, add selected categories to list in `Help` -> `Diagnostic Tools` -> `Debug Log Settings`. <br/>
-A relatively small amount of `TRACE`-level logs is generated as well.
+### Machete file editor
 
+If you are familiar with console equivalent of this tool ([git-machete](https://github.com/VirtusLab/git-machete#git-machete)) then for sure you know what is the `machete` file.
```

Comment:
Hmm TBH I wouldn't bring this up - if someone used the CLI before, they surely know without that sentence as well; and for newcomers this might works slightly against us - giving the impression that some extra prior knowledge is preferred/desired